app/Community/Components/UserProfileMeta.php

```diff
@@ -322,7 +323,7 @@ private function buildRankMetadata(
     private function buildSocialStats(User $user): array
     {
         $userSetRequestInformation = getUserRequestsInformation($user);
-        $numForumPosts = $user->forumPosts()->count();
+        $numForumPosts = $user->forumPosts()->authorized()->viewable(Auth::user())->count();
```

Comment:
This change fixes a bug on the user profile. The current count of forum posts represents _all_ posts made by the user, including the ones that can't be seen due to being unauthorized or locked behind a permissions gate.

The count now reflects the posts that are visible on the `/user/[user]/posts` page.